Pipfile

```diff
@@ -0,0 +1,24 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+click = "*"
+numpy = "*"
+cffi = "*"
+scipy = "*"
+astropy = "*"
+corner = "*"
+pytest = "*"
+pytest-cov = "*"
+pre-commit = "*"
+tox = "*"
+21cmfast = {git = "git://github.com/21cmFAST/21cmFAST"}
+PyYAML = "*"
+21cmmc = {editable = true,path = "."}
```

Comment:
Since this is in the non-dev section, shouldn't 21CMMC be installed as not editable?

Reply:
It is suggested on `pipenv` page to install all version-controlled packages as editable. I don't fully understand why, it has something to do with dependency checking. So `21cmfast` should also be installed as editable.

Reply:
OK fair enough then :-)